### ISSUE TITLE
Support Gradle 5+

### ISSUE THREAD

**Issue by ChristianMurphy** [Jasig/uPortal#1472]
**Is your feature request related to a problem? Please describe.**

Gradle 5 adds support for Java 11 LTS.
Getting Gradle 5+ supported will unblock uPortal's path toward adopting Java 11 LTS.

**Describe the solution you'd like**
Build system that remains as close to the current setup as possible, but with Gradle 5.

**Describe alternatives you've considered**
Trying to run use Java 8 to run Gradle, and have Gradle try to compile with a separate JDK 11.

**Additional context**
This could then be merged into the ongoing #1007 work.

**Comment by ChristianMurphy** [comment]
https://docs.gradle.org/5.1-rc-1/userguide/upgrading_version_4.html

**Comment by ChristianMurphy** [comment]
resolved by #1501
